packages/myst-cli/src/process/myst/directives.ts

```diff
@@ -378,6 +378,47 @@ const TabSet: IDirective = {
   hast: (h, node) => h(node, 'div', { class: 'margin' }),
 };
 
+const TabItem: IDirective = {
+  myst: class TabItem extends Directive {
+    public required_arguments = 1;
+
+    public optional_arguments = 0;
+
+    public final_argument_whitespace = true;
+
+    public has_content = true;
+
+    public option_spec = {
+      sync: directiveOptions.unchanged,
+      selected: directiveOptions.flag,
```

Comment:
This line is the only change in this file. Moved it up closer to the `tab-set` definition.

Reply:
👍 Makes sense aligning with sphinx